src/utilities.js

```diff
@@ -35,3 +35,20 @@ export const getBabelLoader = (config, isOutsideOfApp) => {
   }
   return babelLoader;
 };
+
+export const tap = (message, options) => (config) => {
+  const print = []
+  print.push(`Tapping the configuration`)
+  if (message) print.push(message)
+  print.push(JSON.stringify(config))
+
+  const fileDest = options && options.dest
+  if (fileDest) {
+    const fs = require('fs')
+    fs.appendFile(fileDest, `${print.join('\n')}\n`)
+  } else {
+    print.map(sentence => console.log(sentence))
```

Comment:
Makes more sense as a `forEach` since we're not really mapping anything here, but it doesn't matter :P

Reply:
I changed it to use a `forEach`.